### PR TITLE
[MRG][ar] Translation of tutorials/keras/overfit_and_underfit tutorial into Arabic

### PR DESCRIPTION
**Pull request by mohamed-ali**
Translation of https://www.tensorflow.org/tutorials/keras/overfit_and_underfit into Arabic.

### CODE REVIEW

**Comment by mohamed-ali** [comment]
@NourShawky @baheytharwat @tayciryahmed Could you review this notebook? 
Thanks.

File: site/ar/tutorials/keras/overfit_and_underfit.ipynb [hunk 1]
```diff
@@ -0,0 +1,1953 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "KX3Voac-FEO4"
+      },
+      "source": [
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "يمكنك عرض [نتائج إحدى دورات التدريب السابقة](https://tensorboard.dev/experiment/fGInKDo8TXes1z7HQku9mw/#scalars&_smoothingWeight=0.97) لهذا الدفتر التّفاعلي على موقع [TensorBoard.dev](https://tensorboard.dev/).\n",
```
Comment (tayciryahmed) [inline site/ar/tutorials/keras/overfit_and_underfit.ipynb:1850]: نتائج إحدى تجارب التدريب السابقة

File: site/ar/tutorials/keras/overfit_and_underfit.ipynb [hunk 2]
```diff
@@ -0,0 +1,1953 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "WL8UoOTmGGsL"
+      },
+      "source": [
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "## تثبيت الحزم و تعديل الإعدادات\n",
```
[the remark this answers is not in the document]
Comment (mohamed-ali) [reply site/ar/tutorials/keras/overfit_and_underfit.ipynb:208]: what about this  تجهيز بئية العمل?

File: site/ar/tutorials/keras/overfit_and_underfit.ipynb [hunk 3]
```diff
@@ -0,0 +1,1953 @@
...
+        "id": "bUUHoXb7w-_C"
+      },
+      "source": [
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "استعمال الدّالة\n",
+        "`l2`\n",
+        "بهذا الشكل\n",
+        "`l2(0.001)`\n",
+        "يعني أن كل معامل\n",
```
[the remark this answers is not in the document]
Comment (mohamed-ali) [reply site/ar/tutorials/keras/overfit_and_underfit.ipynb:1510]: There are several coefficient. What you suggest works if there is only one. 

See the original text
 

> l2(0.001) means that every coefficient in the weight matrix of the layer will add 0.001 * weight_coefficient_value**2 to the total loss of the network.

File: site/ar/tutorials/keras/overfit_and_underfit.ipynb [hunk 4]
```diff
@@ -0,0 +1,1953 @@
...
+        "L2.\n",
+        "\n",
+        "يؤدي الضبط بطريقة \n",
+        "L1\n",
+        "إلى دفع الأوزان إلى الصفر حرفيّا و هو ما يُنتج نموذجًا قليل الكثافة - و يسمّى أيضا نموذجا متناثرًا\n",
+        "(sparse model).\n",
+        "وأمّا طريقة الضبط باستعمال\n",
+        "L2\n",
+        "ستُعاقب \n",
+        "  الأوزان دون جعلها متناثرة أو قليلة الكثافة و ذلك لأنّ قيمة هذه \"التكلفة الزائدة\" أو العقوبة ستكون قريبة إلى الصفر للأوزان الصغيرة.\n",
```
[the remark this answers is not in the document]
Comment (mohamed-ali) [reply site/ar/tutorials/keras/overfit_and_underfit.ipynb:1456]: أستعمل عقوبة كمرادف ل
penalty
و تبدو أوضح في هذا السياق

**Comment by mohamed-ali** [comment]
@lamberta This PR is approved. Could you merge it before updating the website, if nothing else needs to be changed? Thanks.
